kubernetes/migration/go.mod

```diff
@@ -0,0 +1,3 @@
+module /Users/srajan/go/src/github.com/coredns/deployment/kubernetes/migration/
```

Comment:
This has you local path in it.  I think it instead should be ...
`module github.com/coredns/deployment/kubernetes/migration/`

Reply:
oh oops